test/parallel/sql/utils/random_array.sql

```diff
@@ -0,0 +1,11 @@
+CREATE OR REPLACE FUNCTION random_int_array(dim integer, min integer, max integer) RETURNS integer[] AS $BODY$
```

Comment:
I feel like this would be useful in regular tests as well. Should be in the utils for regular tests.